app/views/comments/thread.html.erb

```diff
@@ -1,7 +1,14 @@
 <% pingable = get_pingable(@comment_thread) %>
 
+<% if @post.parent.present? %>
+  <details>
+    <summary>Question</summary>
```

Comment:
Not all post types with parents are questions; we can dynamically add others. I'd suggest calling this "Parent post" (or you could fetch the post type and display its name, but "parent" covers all cases).